### PR TITLE
Add new source elvanto

### PR DESCRIPTION
**Pull request by bmcclenahan**

### CODE REVIEW

**Comment by bmcclenahan** [comment]
I've signed it

**Comment by bmcclenahan** [comment]
I've signed a CLA

File: sources/elvanto_API [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"id": "ELVANTO_API",
```
Comment (pfrisella) [inline sources/elvanto_API]:
```suggestion
	"id": "ELVANTO",
```

and also update filename

File: sources/elvanto_API [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	"name": "elvanto-API",
```
Comment (pfrisella) [inline sources/elvanto_API]:
```suggestion
	"name": "Elvanto",
```

File: sources/elvanto_API [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+	"categories": ["CUSTOM"],
+	"organization": "ELVANTO",
+	"iconUrl": "https://www.elvanto.com/resources/branding/?download=icon-png",
+	"sourceUrl": "https://www.elvanto.com/api/",
```
Comment (pfrisella) [inline sources/elvanto_API]:
```suggestion
	"sourceUrl": "https://www.elvanto.com/",
```

**Comment by bmcclenahan** [comment]
I have committed the suggested changes - thanks for the suggestions

**Comment by bmcclenahan** [comment]
confirmed by bruce mcclenahan

**Comment by bmcclenahan** [comment]
the suggested changes have been made
the author consents
please progress the pull request

**Pull request merged**